css/auth.css

```diff
@@ -0,0 +1,168 @@
+/* reset */
+a{
+    color: var(--black);
+}
+
+/* root-styles */
+:root {
+    --grey-color: #f4f4f4;
+    --dark-grey-color: #c4c2c2;
+}
+
+body{
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+body{
+    background-color: var(--grey-color);
+}
+
+/* display-grid */
+.dgrid-fieldset{
+    display: grid;
+    gap: var(--size-5);
+}
+
+.dgrid-section{
+    display: grid;
+    gap: var(--size-13);
+    padding: var(--size-9);
+}
+
+/* display-flex */
+.dflex{
+    justify-content: space-between;
+}
+
+.dflex :last-child{
+    padding-right: 0;
+    text-decoration: underline;
+}
+
+/* header */
+.header{
+    padding: calc(3*var(--size-9)) var(--size-9) 0 var(--size-9);
+}
+
+.nav{
+    padding: var(--size-9);
+    display: grid;
+    align-items: center;
+    grid-template-columns: 1fr auto;
+    gap: var(--size-13) 0;
+}
+
+.txt-logo{
+    font-weight: var(--weight-black);
+    grid-column: 1;
+    font-size: var(--size-10);
+}
+
+.actions{
+    grid-column: 2;
+    font-weight: var(--weight-bold);
+}
+
+.actions > a{
+    text-decoration: underline;
+}
+
+.nav > h1{
+    padding: calc(4*var(--size-9)) 0 0 0;
+    grid-column: 1/-1;
+    font-size: 2.5rem;
+    line-height: 1.2;
+}
+
+/* main */
+.main{
+    padding: 0 var(--size-9) calc(3*var(--size-9)) var(--size-9);
+}
+
+.input-group-text{
+    background:0; 
+    border: 0;
+    width: max-content;
+    font-weight: var(--weight-bold);
+    padding-left: 0;
+}
+
+.input{
+    cursor: initial;
+    padding: var(--size-7) var(--size-9);
+    border: 0.0625rem solid;
+    border-radius: var(--size-3);
+}
+
+.tandc{
```

Comment:
Consider going with better and meaningful classname

Reply:
I changed class name to `signup_conditions`.